ui/src/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+interface FooterButtons {
+  label: string;
+  path: string;
+}
+
+const footerButtons: FooterButtons[] = [
+  { label: "Q&A", path: "/QA" },
+];
+export function Footer() {
+  return (
+    <footer className="footer">
+      <a className="footer__img" href="/home">
+        <img src="peachy.png" width="55" height="55"/>
+      </a>
+      {footerButtons.map((button) => {
+        return (
+          <a className="footer__button" href={button.path} key={button.label}>
```

Comment:
NavBar looks good, but have the buttons be a different style than the navbar buttons. Up to you how you want to style them. But have them be unique